README.md

```diff
@@ -74,6 +80,10 @@ if warnThreshold.DoesViolate(3.6) {
 
 ## Perfdata
 
+The Perfdata object represents Nagios-style performance data that relates to the actual execution of a host or service check. 
+
+https://nagios-plugins.org/doc/guidelines.html#AEN200
```

Comment:
I prefer this https://github.com/monitoring-plugins/monitoring-plugin-guidelines/blob/main/monitoring_plugins_interface/03.Output.md#performance-data

Reply:
Done